### ISSUE TITLE
Got Permaban watch out guz

### ISSUE THREAD

**Issue by Miiwii** [finndev/PokeBuddy#933]
Hey guyz got a permaban today.
I wrote a mail to the support and they said me (nintendo not niantic ) that i got permabaned for boting!

So watch out guyz! Niantic is watching you now

**Comment by jacurtwong** [comment]
so now u cant login to your account?

**Comment by vicmacky** [comment]
yes, what do you mean? no login at all?

**Comment by mufclaw** [comment]
i can get on but everytime i go to pokestop it says try again later and i cant get myself unsoftbanned doing 40x spins

**Comment by Miiwii** [comment]
Yes after i choose my google account there is no loading stick and nothing is happening. after 10 seconds i got a message with data not founds from server and the support told me that s a permaban

**Comment by vicmacky** [comment]
that's really odd, so some accounts are permabanned and others are softbanned with no possible unban with pokespin... maybe they changed the way to unban yourself

**Comment by mufclaw** [comment]
im not sure mines been like it for 4 hours now other accounts can loot the stops but on my main account it just says try again later and wont unsoftban

**Comment by Deuxgod** [comment]
What has Nintendo to do with Pokemon Go other than licensing the name? Nintendo has no part on this game, no authority to issue bans or even gives support. It's Niantic the one and only who can give you support! If you want to troll at least get your info right.

**Comment by Miiwii** [comment]
I dont want to troll someone. I just want to warn you here because in fact there are permabans now,
Nintendo support answered me so i think they have informations about whats going on.
If you dont believe me then ignore this warning and you ll see whats happening then :)

**Comment by Yalmarr** [comment]
He aint trolling @Deuxgod 
More and more people are having these problems.

**Comment by Deuxgod** [comment]
I'm having the same issue also, I know that something is up, but it doesn't add up that Nintendo would give support when they are not even related with the project. Plus if you check server status the US ones are down and being hit like hell. There's reports of some people that had the issue getting in again, so just wait and let's see how it plays out.

**Comment by Miiwii** [comment]
@Deuxgod i hope you are right. but im playing on the EU Realm and the Server is online and i cant log in.

**Comment by Deuxgod** [comment]
http://downdetector.com/status/pokemon-go/map/

**Comment by Rab89** [comment]
proof or gtfo nintendo would have no interference with a game not made by them :laughing:

**Comment by JewBagel** [comment]
i feel like this is more panic than a permabanned and he corrected himself with the nintendo crap lol... the servers are up and fine i was just playing on my phone the bot just doesn't work currently for me at least

**Comment by Rab89** [comment]
yep im having no problems with the app but as far as botting goes im taking a backseat for a while lol

**Comment by JewBagel** [comment]
yeah not sure if to quit while still ahead lol

**Comment by naor173** [comment]
if ur close to level 30 and everything still work fine for you bro.. take a break from bots and stuff i think it will be more sad if you will just get banned.. since niantic start using their power against us

**Comment by JewBagel** [comment]
yeah thats why i want to quit dont want to get permabanned but they might catch us anyway is what im thinking as much as i hope not but these things will come in big waves so we are in the clear for now

**Comment by dodolock** [comment]
@Miiwii do not troll here. Nintendo had nothing to do with this company. The three company worked different part for this game. Nintendo part is to authorize the other two companies to use pokemon super IP. TROLLER  do the research before trolling

**Comment by xReVoLT-iTheme** [comment]
https://github.com/finndev/PokeBuddy/issues/912

**Comment by mufclaw** [comment]
necro .7.0 running fine
